telemetry/client.go

```diff
@@ -283,13 +284,40 @@ func (c *Client) SendFunctionLogs(ctx context.Context, invokedFunctionARN string
 	return nil
 }
 
+// getNewRelicTags adds tags to the logs if NR_TAGS has values
+func getNewRelicTags(common map[string]interface{}) {
+    nrTagsStr := os.Getenv("NR_TAGS")
+    nrDelimiter := os.Getenv("NR_ENV_DELIMITER")
+    if nrDelimiter == "" {
+        nrDelimiter = ";"
+    }
+
+    if nrTagsStr != "" {
+        tags := strings.Split(nrTagsStr, nrDelimiter)
+        nrTags := make(map[string]string)
+        for _, tag := range tags {
+			fmt.Println(tag)
```

Comment:
instead of fmt, can we use util.Debugf() ?

Reply:
Removed `fmt`, in revision, which was used for local debugging.